src/rpc/quorums.cpp

```diff
@@ -1017,21 +1017,23 @@ static UniValue submitchainlock(const JSONRPCRequest& request)
     if (nBlockHeight <= 0) {
         throw JSONRPCError(RPC_INVALID_PARAMETER, "invalid block height");
     }
+    const LLMQContext& llmq_ctx = EnsureLLMQContext(EnsureAnyNodeContext(request.context));
+    auto bestCLHeight = llmq_ctx.clhandler->GetBestChainLock().getHeight();
+    if (nBlockHeight <= bestCLHeight) return bestCLHeight;
```

Comment:
IMO better to return blockHash or a pair of blockHash + blockHeight

@QuantumExplorer why do you want just a height?